utils/vpr_io_place.py

```diff
@@ -41,7 +41,7 @@ def constrain_net(self, net_name, loc, comment=""):
             comment=comment,
         )
 
-    def output_io_place(self, f):
+    def output_io_place(self, f, net_to_block=None):
```

Comment:
https://github.com/SymbiFlow/symbiflow-arch-defs/pull/1011#discussion_r325885181

Reply:
I moved the code that loads block names from the .net file into `load_block_names_from_net_file` in this class with a docstring.